pkg/proxy/userspace/proxier.go

```diff
@@ -304,6 +310,26 @@ func (proxier *Proxier) Shutdown() {
 	close(proxier.stopChan)
 }
 
+func (proxier *Proxier) setInitialized(servicesSynced, endpointsSynced bool) {
```

Comment:
This helper is more confusing than helpful, especially the fact that `setInitialized(true, false)` doesn't mean "set `endpointsSynced` to not initialized".